sale_order_project/views/analytic_account_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_analytic_account_form_sale_order" model="ir.ui.view">
+        <field name="name">account.analytic.account.form.sale.order</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
+        <field name="arch" type="xml">
+            <xpath expr='//div[@name="button_box"]' position='inside'>
+                <button name="%(sale.action_orders)d"
+                        icon="fa-shopping-cart"
```

Comment:
remove `icon` and use `class`

ex.` <i class="fa-shopping-cart" />`

Reply:
i'm reverting that:
![que](https://user-images.githubusercontent.com/5034215/46792300-bb18b500-cd4b-11e8-99ab-19d5191240be.png)